src/query/functions/temporal/aggregation.go

```diff
@@ -85,28 +92,42 @@ func NewAggOp(args []interface{}, optype string) (transform.Params, error) {
 			aggFunc: aggregationFunc,
 		}
 
+		if optype == QuantileType {
+			if len(args) != 2 {
```

Comment:
This is pretty messy; do a similar approach to what's there for holt winters instead